README.md

```diff
@@ -2422,6 +2423,60 @@ You can configure this using `metadata-policy` element for the map configuration
 </hazelcast>
 ```
 
+#### 7.7.1.5 Filtering with Paging Predicates
+
+Hazelcast Python client provides paging for defined predicates. With its `PagingPredicate`, you can get a collection of keys, values, or entries page by page by filtering them with predicates and giving the size of the pages. 
+Also, you can sort the entries by specifying comparators. In this case, the comparator should be either `Portable` or `IdentifiedDataSerializable` 
+and the serialization factory implementations should be registered on the member side. Please note that,
+paging is done on the cluster members. Hence, client only sends a marker comparator to indicate members
+which comparator to use. The comparision logic must be defined on the member side by implementing the 
+`java.util.Comparator<Map.Entry>` interface.
+
+Paging predicates require the objects to be deserialized on the member side from which the collection is retrieved. 
+Therefore, you need to register the serialization factories you use on all the members on which the paging predicates are used.
+See the [Adding User Library to CLASSPATH](#1212-adding-user-library-to-classpath) section for more details.
+
+In the example code below:
+
+- The `is_greater_than_or_equal_to` predicate gets values from the `students` map. 
+This predicate has a filter to retrieve the objects with an `age` greater than or equal to `18`.
+
+- Then a `PagingPredicate` is constructed in which the page size is `5`, so that there are five objects in each page. 
+The first time the values are called creates the first page.
+
+- It gets subsequent pages with the `next_page()` method of `PagingPredicate` and querying the map again with the updated `PagingPredicate`.
+
+```python
+from hazelcast.serialization.predicate import paging, is_greater_than_or_equal_to
+
+...
+
+m = client.get_map("students").blocking()
+predicate = paging(is_greater_than_or_equal_to("age", 18), 5)
+
+# Retrieve the first page
+values = m.values(predicate)
+
+...
+
+# Set up next page
+predicate.next_page()
+
+# Retrieve next page
+values = m.values(predicate)
+```
+
+If a comparator is not specified for `PagingPredicate`, but you want to get a collection of keys or values page by page, 
```

Comment:
nit: does it make sense to stress the fact that the `If a comparator is not specified for `PagingPredicate`` part speaks of the cluster-side class? Python users may be confused otherwise.

Reply:
I believe this part is meant for the cases like `paging(predicate, 5, None)` or `paging(predicate, 5)` when the comparator is not specified on the client-side. Did I get your question wrong?

Reply:
Probably, I misread this part. Thanks for the clarification. Resolving this one.